libraries/helpers.rb

```diff
@@ -477,6 +470,16 @@ def get_metadata_with_token(token, uri)
   metadata
 end
 
+def configure_gc_thresh_values
+  (1..3).each do |i|
+    # Configure gc_thresh values to be consistent with alinux2 default values
+    sysctl "net.ipv4.neigh.default.gc_thresh#{i}" do
+      value node['cluster']['sysctl']['ipv4']["gc_thresh#{i}"]
+      action :apply
+    end
+  end
+end
+
```

Comment:
Please double check if this is required.
It isn't part of my changes. It's likely a leftover of a rebase.